ddmrp_production_equivalent/models/mrp_production.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MrpProduction(models.Model):
+    _inherit = 'mrp.production'
+
+    def _get_product_equivalent(self, bom_line, requested_qty):
+        # get all the other products in the same product category
+        p_obj = self.env['product.product']
+        product_ids = p_obj.search([
```

Comment:
please rename `product_ids` to `products`, the variable doesn't contain ids but records